seedingscript.js

```diff
@@ -0,0 +1,187 @@
+const mysql = require('mysql');
+const faker = require('faker');
+
+let db = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+  database: 'review_data'
+});
+
+let userHoursOnRecord = {};
+
+
+//----------------------------------------
+//Helper functions
+//----------------------------------------
+
+
+let generateReleaseDates = (numGames) => {
+  let releaseDates = {};
+  let startingGameId = 1;
+
+  while (startingGameId <= numGames) {
+    releaseDates[startingGameId] = faker.date.between('2012-01-01', '2021-01-18');
+    startingGameId++;
+  }
+
+  return releaseDates;
+}
+
+let generateArrayOfUserIds = (numUsers) => {
+  let arrayOfUserIds = [];
+
+  while (numUsers > 0) {
+    arrayOfUserIds.unshift(numUsers);
+    numUsers--;
+  }
+
+  return arrayOfUserIds;
+}
+
+let deleteAllFromTable = (tableName) => {
+
+  db.query(`DELETE FROM ${tableName};`, (err) => {
+    if (err) {
+      console.log(`Error deleting all from table '${tableName}': `, err);
+      return;
+    }
+  });
+  console.log(`All rows from table ${tableName} successfully deleted`);
+
+}
+
+
+//----------------------------------------
+//Seeding functions
+//----------------------------------------
+
+
+let seedUsers = (numUsers) => {
+
+  if (numUsers === 0) {
+    console.log('No users to seed');
+    return;
+  }
+
+  deleteAllFromTable('users');
+
+  let startingUserId = 1;
+
+  while (startingUserId <= numUsers) {
+
+    let data = {
```

Comment:
I see you've already figured out the quickest way to set a max at a variable of 10 and still include 0. Nicely done.